boards/arm/mimxrt1015_evk/mimxrt1015_evk.dts

```diff
@@ -53,6 +53,7 @@ arduino_serial: &uart4 {};
 	reg = <0x402a8000 0x4000>, <0x60000000 0x1000000>;
 	at25sf128a: at25sf128a@0 {
 		compatible = "adesto,at25sf128a", "jedec,spi-nor";
+		size = <134217728>;
```

Comment:
Isn't it possible to put number in hexadecimal representation here? I believe it will by natural.

Reply:
It really isn't, though that's what I used to do.  The datasheets tend to provide them in decimal, and for normal humans it's a lot easier to understand that `134217728` is 128 Mbit and `67108864` is 64 Mbit than to try to understand what `0x8000000` and `0x400000` mean.

Also the majority of nodes that weren't added by me use decimal, so I decided to follow the common path.

Reply:
I can accept that, better align to majority in this case